### PR TITLE
Prevent Windows make clean from removing OpenRA.Platforms.Default.dll.config

### PR DESCRIPTION
**Pull request by reaperrr**
`OpenRA.Platforms.Default.dll.config` is part of the repo.
Fixes that `OpenRA.Platforms.Default.dll.config` gets removed every time you run `make clean` on Windows.

Linux makefile already excludes the file from removal.

### CODE REVIEW

File: make.ps1 [hunk 1]
```diff
@@ -38,7 +38,6 @@ function Clean-Command
 	{
 		$proc = Start-Process $msBuild $msBuildArguments -NoNewWindow -PassThru -Wait
 		rm *.dll
-		rm *.dll.config
```
Comment (pchote) [inline make.ps1]: Stack overflow suggests something like this should work:
```powershell
Get-ChildItem *.dll.config -exclude OpenRA.Platforms.Default.dll.config | Remove-Item
```

Comment (reaperrr) [reply make.ps1]: Looks like it does.

**Comment by reaperrr** [comment]
Updated.

**Review by pchote — APPROVED** [review]

**Comment by reaperrr** [comment]
If this doesn't get another review by the end of the weekend, I'll go ahead and merge it.

**Review by abcdefg30 — APPROVED** [review]

**Pull request merged**